loglist2/loglist2.go

```diff
@@ -136,6 +136,59 @@ type TreeHead struct {
 	TreeSize int64 `json:"tree_size"`
 }
 
+// String method returns printable name of the state.
+func (ls *LogStates) String() string {
+	names := [...]string{
+		"Empty",
+		"Pending",
+		"Qualified",
+		"Usable",
+		"Frozen",
+		"Retired",
+		"Rejected"}
+	switch {
+	case ls == nil:
+		return names[0]
+	case ls.Pending != nil:
+		return names[1]
+	case ls.Qualified != nil:
+		return names[2]
+	case ls.Usable != nil:
+		return names[3]
+	case ls.Frozen != nil:
+		return names[4]
+	case ls.Retired != nil:
+		return names[5]
+	case ls.Rejected != nil:
+		return names[6]
+	default:
+		return names[0]
+	}
+}
+
+// Active picks the set-up state. If multiple states are set (not expected) picks one of them.
```

Comment:
I think we can assume that the log list conforms to the schema, which requires that only one state is set.

Reply:
Ack.